src/MetaDataItem.py

```diff
@@ -1,2 +1,25 @@
+import json
+
 class MetaDataItem:
-    pass
+    def __init__(self, id, title, url, collision_type, description, location):
+        self.id = id
+        self.title = title
+        self.url = url
+        self.collision_type = collision_type
+        self.description = description
+        self.location = location
+        self.obj = {
+            'title': self.title,
+            'url': self.url,
+            'collision_type': self.collision_type,
+            'description': self.description,
+            'location': self.location
+        }
+
+    def to_json_str(self) -> str:
```

Comment:
can you please follow the requirements and name this `to_json`
```suggestion
    def to_json(self) -> str:
```

Reply:
ok

Reply:
Without qualifying it with `str` it becomes ambiguous. This could be returning the JSON object and not the string.